docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-dagster.md

```diff
@@ -38,120 +38,221 @@ by signing up for the trial.
-`dlt`", please refer to [article](https://dagster.io/blog/dagster-dlt). The author offers a
-detailed overview and steps for ingesting GitHub issue data from a repository and storing it in
-BigQuery. You can use a similar approach to build your pipelines.
+
+    For more information, please refer to
+    [Dagster’s documentation.](https://docs.dagster.io/_apidocs/libraries/dagster-embedded-elt#dagster_embedded_elt.dlt.dlt_assets)
+
+ 1. Create the Definitions object.
+
+    The last step is to include the assets and resource in a [Definitions](https://docs.dagster.io/_apidocs/definitions#dagster.Definitions) object (`github_source/definitions.py`). This enables Dagster tools to load everything we have defined:
+
+     ```py
+     import assets
+     from dagster import Definitions, load_assets_from_modules
+     from dagster_embedded_elt.dlt import DagsterDltResource
+
+     dlt_resource = DagsterDltResource()
+     all_assets = load_assets_from_modules([assets])
+
+     defs = Definitions(
+         assets=all_assets,
+         resources={
+             "dlt": dlt_resource,
+         },
+     )
+     ```
+
+1. Run the web server locally:
+    1. Install the necessary dependencies using the following command:
+
+       ```sh
+       pip install -e ".[dev]"
+       ```
+
+       We use -e to install dependencies in [editable mode](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs). This allows changes to be automatically applied when we modify the code.
+
+    2. Run the project:
+
+       ```sh
+       dagster dev
+       ```
+
+    3. Navigate to localhost:3000 in your web browser to access the Dagster UI.
+
+       ![image](https://github.com/user-attachments/assets/97b74b86-df94-47e5-8ae2-de7cc47f56d8)
+
+1. Run the pipeline.
+
+   Now that you have a running instance of Dagster, we can run our data pipeline.
+
+   To run the pipeline, go to **Assets** and click the **Materialize** button in the top right. In Dagster, materialization refers to executing the code associated with an asset to produce an output.
+
+   ![image](https://github.com/user-attachments/assets/79416fb7-8362-4640-b205-e59aa7ac785c)
+
+   You will see the following logs in your command line:
+
+   ![image](https://github.com/user-attachments/assets/f0e3bec8-f702-46a6-b69f-194a1dacf625)
+
+   Want to see real-world examples of dlt in production? Check out how dlt is used internally at Dagster in the [Dagster Open Platform](https://github.com/dagster-io/dagster-open-platform) project.
+
+
+:::info
+For a complete picture of Dagster's integration with dlt, please refer to their [documentation](https://docs.dagster.io/integrations/embedded-elt/dlt). This documentation offers a detailed overview and steps for ingesting GitHub data and storing it in Snowflake. You can use a similar approach to build your pipelines.
 :::
 
+### Frequently Asked Questions
+- **Can I remove the generated .dlt folder?**
```

Comment:
```suggestion
- **Can I remove the generated `.dlt` folder with `secrets.toml` and `config.toml` files?**
```

because we have another .dlt folder in the home directory, and users shouldn't remove it:)